testing/bdd/features/timing/delay.feature

```diff
@@ -20,29 +20,29 @@ Feature: Delay of a document sequence
     # Negative value should really throw an exception
     | 00:00:10.0 | -00:00:00  | 00:00:10.0         |  
 
-#TODO: test that the order of documents in the input and output sequence is the same (FIFO) 
+#TODO: test that the order of documents in the input and output sequence is the same (FIFO)
 
 
   #TBC: this is a change to the spec (passive delay node)
   @skip
-  Scenario: Implicitly timed document unchanged sequence identifier 
+  Scenario: BufferDelayNode, unchanged sequence identifier
     Given an xml file <xml_file>
     And the document is generated
     And sequence identifier <sequence_id_1>
     When the delay node delays it by <delay>
     Then the delayed document has <sequence_id_2>
 
     Examples:
-    | sequence_id_1 | delay      | sequence_id_2 |  
-    | 1             | 00:00:02.0 | 1             |  
-    | xxx           | 00:00:00   | xxx           |  
-    | 99999999999   | 99:00:00   | 99999999999   | 
+    | sequence_id_1 | delay      | sequence_id_2 |
+    | 1             | 00:00:02.0 | 1             |
+    | xxx           | 00:00:00   | xxx           |
+    | 99999999999   | 99:00:00   | 99999999999   |
     # No document generated for negative delay
     | 1             | -00:00:01  |               |
```

Comment:
This last line doesn't feel right here. It's checking a different conformance requirement that may not actually exist.